Documentation~/Blender.md

```diff
@@ -94,3 +95,36 @@ The following is a list of supported nodes and how they are handled.
 | Normal Map | Normal strength is exported. The normal map needs to be in tangent space. |
 | Image Texture | Images are exported but not the texture coordinates. Unity uses UV0. |
 | Displacement | Height and scale is exported. |
+
+
+## Material baking
+
+The basic material sync mode only supports bsdf inputs that are textures or constant values.
+MeshSync has functionality to create baked material copies for each object to allow syncing of procedural materials.
+
+|**Option** |**Description** |
+|:---     |:---|
+| Objects to bake | Bakes all objects in the scene or only selected objects (including hidden objects). |
+| Material channels to bake | Choose which channels should be baked. |
+| Generate UVs | Off: Uses UVs on the object, If needed: Generates UVs if there are no usable UVs on the object, Always: Always generates UVs for baking, even if there are existing UVs. NOTE: These settings can be destructive to existing UVs. |
+| Apply modifiers | Applies all modifiers on the object to ensure the UV coordinates are correct for baking. |
```

Comment:
Should we warn that this is not reversible?

Reply:
I've added it.